src/app/routes/request-routes.tsx

```diff
@@ -0,0 +1,67 @@
+import { Suspense } from 'react';
+import { Route } from 'react-router-dom';
+
+import { RouteUrls } from '@shared/route-urls';
+
+import { BroadcastErrorDrawer } from '@app/components/broadcast-error-drawer/broadcast-error-drawer';
+import { EditNonceDrawer } from '@app/features/edit-nonce-drawer/edit-nonce-drawer';
+import { ledgerStacksMessageSigningRoutes } from '@app/features/ledger/flows/stacks-message-signing/ledger-stacks-sign-msg.routes';
+import { ledgerStacksTxSigningRoutes } from '@app/features/ledger/flows/stacks-tx-signing/ledger-sign-tx.routes';
+import { PsbtRequest } from '@app/pages/psbt-request/psbt-request';
+import { StacksMessageSigningRequest } from '@app/pages/stacks-message-signing-request/stacks-message-signing-request';
+import { TransactionRequest } from '@app/pages/transaction-request/transaction-request';
+import { ProfileUpdateRequest } from '@app/pages/update-profile-request/update-profile-request';
+import { AccountGate } from '@app/routes/account-gate';
+import { SuspenseLoadingSpinner } from '@app/routes/app-routes';
+
+// #4028: TODO These are labelled legacy, maybe we can just remove them?
+export const legacyRequestRoutes = (
```

Comment:
Do we need these `legacy` routes?

Reply:
Yes, these are nec to keep for requests not using new rpc methods.